apps/src/applab/setPropertyDropdown.js

```diff
@@ -89,7 +92,10 @@ PROPERTIES[ElementType.BUTTON] = {
     'textAlign',
     'image',
     'iconColor',
-    'hidden'
+    'hidden',
+    'borderWidth',
+    'borderColor',
+    'borderRadius'
```

Comment:
Yes, it is possible - although it may not always be the case that what we surface in the code editor dropdown is what we also want to surface in design mode. But most of the time, that should be the case.